recipe/meta.yaml

```diff
@@ -49,10 +57,17 @@ outputs:
         - {{ pin_subpackage('libtiledbsoma', exact=True) }}
         - anndata
         - pandas
-        - pyarrow
-        - scanpy
+        - pyarrow 9.0.*
         - scipy
-        - spdlog
+        - anndata
+        - tiledb-py 0.21.*
+        - typing-extensions
+        - numba
+        - attrs
+        - somacore 1.0.*
+        - scanpy 1.9.*
```

Comment:
`python` needs to be explicitly added to the run requirements. That is why the current binaries of the Python client don't have build strings that start with `py`